CHANGELOG.unreleased.md

```diff
@@ -16,6 +16,7 @@ Section headings should be at level 3 (e.g. `### Added`).
 ### Added
 
 - The automations API now supports creating and editing automations that trigger on run states (@tonyyli-wandb in https://github.com/wandb/wandb/pull/10848)
+- Apply http headers specified using `WANDB_X_EXTRA_HTTP_HEADERS` for file uploads using presigned url. (@pingleiwandb in https://github.com/wandb/wandb/pull/10761)
```

Comment:
This PR also affects downloads, doesn't it? And users don't know about presigned URLs, I think.

This should probably say something like "File uploads and downloads now respect `WANDB_X_EXTRA_HTTP_HEADERS`".

Except, I don't think that's accurate. I don't think we add headers to HTTP requests for S3 for instance. How would you concisely describe the expected semantics of `WANDB_X_EXTRA_HTTP_HEADERS`?